app/frontend/state.ts

```diff
@@ -142,74 +160,95 @@ const initialState: State = {
     title: 'Wrong mnemonic',
     hint: 'Hint: Ensure that your mnemonic is without mistake.',
   },
+  currentTab: 'wallet-info',
```

Comment:
I believe this isn't used anywhere, I'd remove it

Reply:
Will do.